src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_time_field/__tests__/step_time_field.test.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { StepTimeField } from '../step_time_field';
+
+jest.mock('../components/header', () => ({ Header: 'Header' }));
+jest.mock('../components/time_field', () => ({ TimeField: 'TimeField' }));
+jest.mock('../components/advanced_options', () => ({ AdvancedOptions: 'AdvancedOptions' }));
+jest.mock('../../../lib/extract_time_fields', () => ({
+  extractTimeFields: fields => fields,
+}));
+
+const noop = () => {};
+const indexPatternsService = {
+  fieldsFetcher: {
+    fetchForWildcard: noop,
+  }
+};
+
+describe('StepTimeField', () => {
+  it('should render normally', () => {
+    const component = shallow(
+      <StepTimeField
+        indexPattern="ki*"
+        indexPatternsService={indexPatternsService}
+        goToPreviousStep={noop}
+        createIndexPattern={noop}
+      />
+    );
+
+    expect(component).toMatchSnapshot();
```

Comment:
The snapshot for this test is just about at the threshold for readability, I'd wager. Maybe instead of using snapshots, we can use `find` and `data-test-subj` selectors to make specific assertions about parts of the UI? This is how some other teams are writing Jest tests for complex, highly-composed components like this one.

Reply:
Addressed: https://github.com/elastic/kibana/pull/16129/commits/7e49fda1c7c2e2dc18c543229478e4e31cb2c602